datamade/la-metro-dashboard
# Add DAG to compile packet

## Overview

This PR adds a DAG to compile packets based on the given identifier and document links. The DAG runs this image: https://github.com/datamade/councilmatic-document-merger

@fgregg Could you leave comments about the document merger image in this PR, too?

### Checklist


#### TODO

- [x] Address outstanding merger issues
- [ ] Implement logging

## Testing Instructions

See https://github.com/datamade/la-metro-councilmatic/pull/766 for testing log.

Related to https://github.com/datamade/la-metro-councilmatic/issues/738
